doc/Language/typesystem.pod6

```diff
@@ -258,6 +258,49 @@ you must use private attribute notation instead.
     C.new.say; C.new('answer').say;
     # OUTPUT: «C.new(attr => 42)␤
     #          C.new(attr => "answer")␤»
+    
+=head4 submethod TWEAK
+
+The L<submethod|/type/Submethod> C<TWEAK> is (indirectly) called by
+L<.bless|/type/Mu#method_bless>. It
+is meant to set private and public attributes of a class after the BUILD submethod has been called. The default
+constructor L<.new|/type/Mu#method_new> defined in C<Mu> is the method that
+invokes it. Given that public accessor methods are not available in C<TWEAK>,
```

Comment:
Not clear this is that way, or that actually means something. It's a phaser, called after BUILD. Whether it's called from Mu.new or not does not seem too relevant.
Also, are you sure public methods are not available? Can you provide an example?

Reply:
I think there is confusion here. Either you or me. I think: BUILD phaser and object submethod BUILD (and TWEAK) are not the same.